### ISSUE TITLE
Missing CHANGELOG.rst in release

### ISSUE THREAD

**Issue by NicolaiSoeborg** [jurismarches/luqum#20]
It seems like you forgot to include `CHANGELOG.rst` in the newest release, which means https://github.com/jurismarches/luqum/blob/master/setup.py#L9 will fail and the package can't be installed.

```
$ sudo -H pip3 install luqum
Collecting luqum
  Using cached luqum-0.6.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-vlx183/luqum/setup.py", line 9, in <module>
        with open('CHANGELOG.rst', 'r') as f:
    IOError: [Errno 2] No such file or directory: 'CHANGELOG.rst'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-vlx183/luqum/
```

**Comment by alexgarel** [comment]
Hi, thanks for the report, I'll look into this right now.

**Comment by alexgarel** [comment]
It's released as 0.6.1. I removed the 0.6.0 source file from pypi to avoid problems.
